packages/clay-color-picker/src/index.tsx

```diff
@@ -202,7 +202,6 @@ const ColorPicker: React.FunctionComponent<Props> = ({
 										onChange={onValueChange}
 										onColorsChange={onColorsChange}
 										spritemap={spritemap}
-										value={value}
```

Comment:
Do we have any reason why we removed this? I can not tell if the reason was this but we are causing a regression, now I can no longer type in the input in the footer within the dropdown.

Reply:
Good catch. I had to make sure and check the activeElement for this. The reason I removed the `value` prop is because it isn't necessarily accurate because it depends which index is selected from the custom colors.

Reply:
Makes sense, thanks for the information.